nemo_curator/utils/distributed_utils.py

```diff
@@ -388,7 +493,8 @@ def read_data(
     input_files: Union[str, List[str]],
     file_type: str = "pickle",
     backend: Literal["cudf", "pandas"] = "cudf",
-    files_per_partition: int = 1,
+    blocksize: Optional[str] = None,
+    files_per_partition: Optional[int] = 1,
     add_filename: bool = False,
     input_meta: Union[str, dict] = None,
     columns: Optional[List[str]] = None,
```

Comment:
Can you elaborate a bit more on a comment from the README: a slowdown when `columns` are selected and the file type is JSON?

Reply:
I haven't benchmarked it, so I can remove that comment, but I remember long ago seeing a noticeable slowdown when you pass in `columns`. But memory is fuzzy now, so I'll remove it, just something to keep in mind.

Reply:
Sounds good, thank you.